misc/module_config.json

```diff
@@ -1,10 +1,12 @@
 {
   "modules": {
     "pass": {
-      "enabled": true
+      "enabled": true, 
+      "timeout": 60
     },
     "nmap": {
-      "enabled": true
+      "enabled": true,
+      "timeout": 60
```

Comment:
why make this change?  When I checked nmap was pretty quick (seconds) -- is there a case where 60 fails?